src/styles/blocks/card.scss

```diff
@@ -0,0 +1,119 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+  padding: 50px 40px;
+
+  @media (min-width: 300px) {
+    grid-template-columns: repeat(1, 200px);
+  }
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+
+  .card {
+    width: 200px;
+    border-radius: 5px;
+    border: 1px solid #f3f3f3;
+    background: $global-white;
+    padding-block: 32px 16px;
+    padding-inline: 16px;
+    font-size: 12px;
+    font-weight: 400;
+    color: $main-text-color;
+
+    &__image {
+      margin-bottom: 40px;
+      text-align: center;
+
+      img {
+        width: 160px;
+        height: 134px;
+        vertical-align: top;
+      }
+    }
+
+    &__title {
+      margin-bottom: 16px;
+
+      &-name {
+        color: $accent-text-color;
+        font-weight: 500;
+        line-height: 18px;
+        margin-bottom: 4px;
+      }
+
+      &-product-code {
+        font-size: 10px;
+        line-height: 14px;
+      }
+    }
+
+    &__review {
+      @include with-two-blocks-content;
+
+      margin-bottom: 24px;
+
+      &-text {
+        color: $accent-text-color;
+        font-size: 10px;
+        line-height: 14px;
+      }
+    }
+
+    &__price {
+      @include with-two-blocks-content;
+
+      margin-bottom: 16px;
+
+      &-name {
+        line-height: 18px;
+      }
+
+      &-amount {
+        color: $accent-text-color;
+        font-size: 16px;
+        font-weight: 700;
+        line-height: 18px;
+      }
+    }
+
+    &__main-button {
+      border-radius: 5px;
+      background-color: $accent-color;
+      color: $global-white;
+      text-align: center;
+      font-size: 14px;
+      font-weight: 700;
+      line-height: 16px;
+      text-transform: uppercase;
+      width: 100%;
+      padding-block: 11px;
+      border: 1px solid $accent-color;
+
+      &:hover {
+        color: $accent-color;
+        background-color: $global-white;
```

Comment:
In this case, you can also add transitions for smooth animation. 
And I highly recommend you add a cursor pointer for elements with actions.